model/task_lifecycle.go

```diff
@@ -583,9 +583,11 @@ func TryDequeueAndAbortCommitQueueVersion(projectRef *ProjectRef, t *task.Task,
 	if p.IsPRMergePatch() {
 		issue = strconv.Itoa(p.GithubPatchData.PRNumber)
 	}
-	removed, err := cq.Remove(issue)
+
+	//question for reviewer:  is it safe to assume that versionExists = true here?
```

Comment:
Tasks are only created for versions--patches are more about the planning phase. You could use `t.Version != ""` instead of just true if you want to be safe, but we use `t.Version` to find the top of the function so we do know that it is populated already.